src/framework/preferences/css/stylus/PrefsEditor.styl

```diff
@@ -236,26 +236,15 @@
     // ON/OFF Switch for checkboxes
     // The container for the toggle, which is also a label for the hidden checkbox.
     .fl-prefsEditor-onoff {
-        .fl-prefsEditor-switch {
-            border-radius:50px;
-            border: 1px solid #776D67;
-            width: 5em;
-            height: 2em;
-            background-color: #E6E6E6;
-            box-shadow: 1em 1.1em 0 0 rgba(250,250,250,0.53) inset;
-            overflow: hidden;
-            vertical-align: middle;
-            display: block;
-            margin-top: 1em;
-            transition-duration: 0.2s;
-            transition-property: padding-left, width, background-color, margin-left;
-            font-size: 1.2em;
-            font-weight: 600;
-            text-transform: lowercase;
+        padding: 1rem;
+
+        label {
+            // so that the header lines up with the other panels
+            margin-top: -1rem;
```

Comment:
Seems a bit odd to have a -1em margin here. Looking at the CSS, there's a 1em padding being added by fl-prefsEditor-onoff. Would it make more sense to put a margin of padding: 0 1rem; on fl-prefsEditor-onoff and remove line 243?